Intersect (Core)/Threading/Threaded`1.cs

```diff
@@ -1,33 +1,32 @@
-﻿using System;
-
-namespace Intersect.Threading
+﻿namespace Intersect.Threading
 {
-
-    /// <inheritdoc />
+    /// <summary>
+    /// Represents a generic base class for creating and managing new threads.
+    /// </summary>
```

Comment:
This part of the documentation should be moved to the base class, and inheritdoc should remain here.